### ISSUE TITLE
[Feature Request] Setting to save all files before running test(s)

### ISSUE THREAD

**Issue by oakesville** [hbenl/vscode-test-explorer#148]
It would be convenient to have an option to save all unsaved files prior to execution.  Either under the existing `onStart` setting or a separate setting.

**Comment by ptournet** [comment]
I agree... I was even baffled to discover that it's not the default behaviour! Therefore a setting for that seems a bare minimum ;)

**Comment by ptournet** [comment]
I've just created a PR for that : #186

**Comment by leoblum** [comment]
@hbenl could you add this PR #186 please?
